project_task_identification/models/project_task.py

```diff
@@ -10,9 +10,14 @@ class ProjectTask(models.Model):
     sy_id = fields.Char(
         compute="_compute_sy_id",
         compute_sudo=True,
+        store=True,
         string="ID",
     )
 
+    @api.depends("name")
```

Comment:
why this?

Reply:
I changed the depends to create_date, what do you think?

Reply:
it's better